resources/creatives/models/creative_updatable.yml

```diff
@@ -0,0 +1,18 @@
+type: object
+
+properties:
+  description:
+    title: Description
+    type: string
+    example: Our updated 4x6 postcard creative
+  from:
+    title: From
+    type: string
+  resource_type:
+    $ref: "../attributes/supported_resource_type.yml"
+  details:
+    title: Details
+    type: object
+  metadata:
```

Comment:
https://github.com/lob/lob-openapi/blob/main/shared/models/metadata.yml

Replace metadata with this (and in any other instances where appropriate).